playwright/tests/script-runner/api.spec.ts

```diff
@@ -26,23 +26,15 @@ test.use({
 })
 
 async function openFile(page, utils, filename) {
-  let half = Math.floor(filename.length / 2)
-  let part1 = filename.substring(0, half)
-  let part2 = filename.substring(half, filename.length)
   await page.locator('[data-test=script-runner-file]').click()
   await page.locator('text=Open File').click()
   await expect(
     page.locator('.v-dialog').getByText('INST2', { exact: true }),
   ).toBeVisible()
-  await utils.sleep(200)
-  await page.locator('[data-test=file-open-save-search]').type(part1)
-  await utils.sleep(200)
-  await page.locator('[data-test=file-open-save-search]').type(part2)
-  await utils.sleep(200)
+  await page.locator('[data-test=file-open-save-search]').type(filename)
```

Comment:
this needs to be `await page.locator('[data-test=file-open-save-search] input').fill(filename)` (missing input)